lib/tapioca/dsl/compilers/active_record_fixtures.rb

```diff
@@ -104,12 +106,53 @@ def method_names_from_eager_fixture_loader
 
         sig { params(mod: RBI::Scope, name: String).void }
         def create_fixture_method(mod, name)
-          mod.create_method(
+          model_class = active_record_models[name] || "T.untyped"
+          mod.create_method_with_sigs(
             name,
-            parameters: [create_rest_param("fixture_names", type: "T.any(String, Symbol)")],
-            return_type: "T.untyped",
+            sigs: [
```

Comment:
fwiw we added a more elegant way to do this in RBI: https://github.com/Shopify/rbi/pull/284.

Reply:
awesome